google/cloud/bigquery/client.py

```diff
@@ -656,10 +705,14 @@ def test_iam_permissions(
         body = {"permissions": permissions}
 
         path = "{}:testIamPermissions".format(table.path)
-
-        response = self._call_api(
-            retry, method="POST", path=path, data=body, timeout=timeout,
-        )
+        span_creator = SpanCreator()
+        attributes = {"path": path, "body": body}
+        with span_creator.create(
+            name="BigQuery.testIAMPolicy", attributes=attributes, client=self
```

Comment:
This name is `BigQuery.testIAMPolicy`, but the function is `test_iam_permissions`. The policy/permissions naming difference could be confusing; is it intentional?